pkg/kubectl/cmd/taint.go

```diff
@@ -366,23 +382,20 @@ func validateNoTaintOverwrites(obj runtime.Object, taints []v1.Taint) error {
 }
 
 // updateTaints updates taints of obj
```

Comment:
can we improve this comment as part of this PR?  the fact that its validating, and reorganizing, ... not sure what 'update' really is doing